src/owncloud/apache_config

```diff
@@ -14,3 +14,18 @@ Alias "/apps" "${SNAP_DATA}/owncloud/apps"
 <IfModule dir_module>
     DirectoryIndex index.html index.php
 </IfModule>
+
+
+
+# MPM Event
+<IfModule event.c>
+    StartServers            2
+    MinSpareThreads         4
+    MaxSpareThreads         32
+    ThreadsPerChild         16
+    ServerLimit             4
+    MaxRequestWorkers       64
+    MaxConnectionsPerChild  0
+</IfModule>
```

Comment:
This snap isn't runing only on the rpi. I assume these numbers would be very low for an amd64 server, for example. I'm not sure there's a good way to keep these numbers specific to the pi, but it might be worth surrounding this with:

```
<If "${SNAP_ARCH} == 'armhf'"">
...
</If>
```

or something. I'm fairly certain all the supported armhf devices would benefit from these numbers, so that seems okay to me. Need to verify that actually works, though.